eventually/static/src/containers/Home.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import FileUpload from './fileUpload/FileUpload';
```

Comment:
Why we need this in import in Home.js?

Reply:
My bad, left it after the test. Removed.